### DISCUSSION TITLE
Client Generator for mac and linux?

### DISCUSSION THREAD

**Discussion by ConGp20** [rustdesk/rustdesk-server-pro#215] [General]
Willl there also be a generator for mac or linux?

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk-server-pro/discussions/217

**Comment by rustdesk** [comment]
Mac is ready, https://twitter.com/rustdesk/status/1777180459429986787

**Comment by rustdesk** [comment]
MSI and Android ready now, Ready now, https://twitter.com/rustdesk/status/1782250852574769663

**Comment by rustdesk** [comment]
Linux is ready now, https://twitter.com/rustdesk/status/1788905463678951787
